### PR TITLE
skaffold v0.1.0 (new formula)

### PR DESCRIPTION
**Pull request by r2d4**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/skaffold.rb [hunk 1]
```diff
@@ -0,0 +1,26 @@
+class Skaffold < Formula
+  desc "Easy and Repeatable Kubernetes Development"
+  homepage "https://github.com/GoogleCloudPlatform/skaffold"
+  url "https://github.com/GoogleCloudPlatform/skaffold.git",
+    :tag => "v0.1.0",
```
Comment (fxcoudert) [inline Formula/skaffold.rb]: We prefer building from a source tarball, to enable Homebrew's caching mechanisms

Comment (fxcoudert) [reply Formula/skaffold.rb]: Version 0.1.0 is marked as “pre-release”. Homebrew's core policy is to only ship stable releases: https://docs.brew.sh/Acceptable-Formulae.html

File: Formula/skaffold.rb [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    dir = buildpath/"src/github.com/GoogleCloudPlatform/skaffold"
+    dir.install buildpath.children - [buildpath/".brew_home"]
+    cd dir do
+      system "make"
+      bin.install "out/skaffold"
+    end
+  end
+
+  test do
+    assert_match "e265bb780f63894f649b8fb1ac53f8aaf89573b2", shell_output("#{bin}/skaffold version --output {{.GitCommit}}")
```
Comment (fxcoudert) [inline Formula/skaffold.rb]: We probably don't want a test that changes at every version

**Review by fxcoudert — DISMISSED** [review]

File: Formula/skaffold.rb (hunk 2 above)
Comment (r2d4) [reply Formula/skaffold.rb]: I've removed the tests that are version dependent.

File: Formula/skaffold.rb (hunk 1 above)
Comment (r2d4) [reply Formula/skaffold.rb]: Our build relies on information in the git repository. We could change our build process add them manually here, at the expense of complicating the brew build.

Pre-release didn't really mean anything to us so I've removed it from the release.

**Comment by r2d4** [comment]
@fxcoudert this should be ready for another look!

**Comment by fxcoudert** [comment]
Thanks @r2d4 for the pull request! And congratulations on your first contribution to Homebrew 🎊

**Comment by r2d4** [comment]
Thanks for the quick review and feedback :+1: !

**Pull request closed without merging**